package-drone/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -28,7 +28,7 @@ sp_cleanup.make_type_abstract_if_missing_method=false
 sp_cleanup.make_variable_declarations_final=false
 sp_cleanup.never_use_blocks=false
 sp_cleanup.never_use_parentheses_in_expressions=true
-sp_cleanup.on_save_use_additional_actions=true
+sp_cleanup.on_save_use_additional_actions=false
```

Comment:
Actually not by mistake. "Remove trailing whitespace" and "Remove redundant type arguments" kept reformatting unrelated code.
-> Re-enabled and changed the settings for whitespace and type arguments.